packages/cli/src/wizard/wizard.js

```diff
@@ -26,8 +34,7 @@ async function runNewProjectWizard(options) {
       type: 'input',
       name: 'serverBaseUrl',
       message: 'What is the URL of your LHCI server?',
-      when: () => !options.serverBaseUrl,
```

Comment:
why present the prompt when we already have it configured?

Reply:
I remove that, to give the user feedback what kind of url was used.
So the User see what happened and can press enter to accept that.
If that's not the way you like I can bring the `when` check back.

Reply:
Alright works for me 👍